docs/output.md

```diff
@@ -1,43 +1,183 @@
 # nf-core/cageseq: Output
 
-This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
+This document describes the output produced by the pipeline. The plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
 
-<!-- TODO nf-core: Write this documentation describing your workflow's output -->
+The directories listed below will be created in the results directory after the pipeline has finished. All paths are relative to the top-level results directory.
 
 ## Pipeline overview
 
 The pipeline is built using [Nextflow](https://www.nextflow.io/)
 and processes data using the following steps:
 
-* [FastQC](#fastqc) - read quality control
```

Comment:
I think you lost the automated TOC

Reply:
Not needed anymore, since we create the ToC from the rendered HTML.

Reply:
I see, I'm not up to date at all with this new system